src/main/java/spring/intro/Main.java

```diff
@@ -0,0 +1,17 @@
+package spring.intro;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import spring.intro.config.AppConfig;
+import spring.intro.model.User;
+import spring.intro.service.UserService;
+
+public class Main {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
+        UserService userService = context.getBean(UserService.class);
+        userService.add(new User("user1", "111"));
+        userService.add(new User("user2", "222"));
+        userService.add(new User("user3", "333"));
+        System.out.println(userService.listUsers());
```

Comment:
does it work correctly?

Reply:
well, yes, I can see the list and all its components in appropriate way.